### ISSUE TITLE
Attempts to spawn powershell in Ubuntu bash

### ISSUE THREAD

**Issue by prescience-data** [microsoft/inshellisense#28]
**Describe the bug**
Crashes on fresh install on Ubuntu when using bash.

**To Reproduce**
1. `volta install @microsoft/inshellisense`
2. `inshellisense bind`
3. `node:internal/errors:496` -> `Error: spawn powershell ENOENT` 

**Expected behavior**
I assume it should call the bash version?

**Environment**
 - OS:Ubuntu 22.04
 - Output of `inshellisense --version`: n/a 
 - Nodejs Version: v18.18.1

**Additional context**
![image](https://github.com/microsoft/inshellisense/assets/65471523/3aa1ed81-ad43-4504-ad81-079cf8cb0f1f)

**Comment by agluszak** [comment]
Same output for fish on ubuntu

**Comment by mhb8898** [comment]
same on mac os x

**Comment by agluszak** [comment]
the new update hasn't fixed it @cpendery :(

```
andrzej@MUNIT-521 ~> inshellisense bind
Select your desired shell for keybinding creation
Loading...
node:internal/errors:497
    ErrorCaptureStackTrace(err);
    ^

Error: spawn powershell ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn powershell',
  path: 'powershell',
  spawnargs: [ '-c', 'echo $profile' ],
  cmd: 'echo $profile',
  stdout: '',
  stderr: ''
}

Node.js v20.8.1
andrzej@MUNIT-521 ~ [1]> inshellisense --version
0.0.1-rc.2
```

**Comment by cpendery** [comment]
> the new update hasn't fixed it @cpendery :(
> 
> ```
> andrzej@MUNIT-521 ~> inshellisense bind
> Select your desired shell for keybinding creation
> Loading...
> node:internal/errors:497
>     ErrorCaptureStackTrace(err);
>     ^
> 
> Error: spawn powershell ENOENT
>     at ChildProcess._handle.onexit (node:internal/child_process:286:19)
>     at onErrorNT (node:internal/child_process:484:16)
>     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
>   errno: -2,
>   code: 'ENOENT',
>   syscall: 'spawn powershell',
>   path: 'powershell',
>   spawnargs: [ '-c', 'echo $profile' ],
>   cmd: 'echo $profile',
>   stdout: '',
>   stderr: ''
> }
> 
> Node.js v20.8.1
> andrzej@MUNIT-521 ~ [1]> inshellisense --version
> 0.0.1-rc.2
> ```

@agluszak it'll be fixed in `0.0.1-rc.3` which hasn't been released yet
